src/middleware/express.ts

```diff
@@ -0,0 +1,28 @@
+import * as express from 'express';
+import renderVoyagerMiddlewarePage, { VoyagerMiddlewareOptions } from './render-voyager-middleware-page';
+
+const { version } = require('../package.json');
```

Comment:
I prefer to keep render-voyager-middleware-page.ts without dependecies. moreover
if you propose to export it from index.js , in my opinion, it makes even more sense not to move dependency inside
> It makes sense to also export renderVoyagerMiddlewarePage function